src/auth-service/utils/request.js

```diff
@@ -40,12 +40,18 @@ const request = {
       await validationsUtil.checkEmailExistenceUsingKickbox(email, (value) => {
         if (value.success == false) {
           const errors = value.errors ? value.errors : "";
-          callback({
+          logObject("the validation checks results", {
             success: false,
```

Comment:
addressing the mailing server error. 
The tool we used was throwing 403 because our credit ($$) on it is down.